code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -379,6 +379,9 @@ emp_act
 			return
 
 		var/obj/item/organ/external/affecting = get_organ(zone)
+		if(!affecting)
+			visible_message("\blue \The [O] misses [src] narrowly!")

```

Comment:
Strongly recommend changing the grammar on this to something like "the [O] wizzes/zooms/whooshes/etc. past [src]'s  missing [missing limb]!"

Or something like that; this is the exact grammar we utilized when attacks (projectiles or melee) could miss; I'd rather not have players think this mechanic is back in.